src/program/lwaftr/tests/subcommands/monitor_test.py

```diff
@@ -23,7 +24,7 @@ class TestMonitor(BaseTestCase):
         str(SNABB_CMD), 'lwaftr', 'run',
         '--name', DAEMON_PROC_NAME,
         '--bench-file', '/dev/null',
-        '--conf', str(DATA_DIR / 'icmp_on_fail.conf'),
+        '--conf', CONFIG_PATH,
```

Comment:
Do we still need to make these changes if the `snabb lwaftr run` is handling setting the PCI devices on incoming configurations?